products/compute/examples/chef/address.rb

```diff
@@ -24,7 +24,7 @@
 
 gcompute_address <%= example_resource_name('test1') -%> do
   action :create
-  region <%= example_resource_name('us-west1') %>
+  region <%= example_resource_name('some-region') %>
```

Comment:
The best route is going to be to substitute `example_resource_name('some-region')` for just 'us-west1' (no need for the example_resource_name function at all)

Reply:
So the test harness will put them in the us-west1 region? I thought that was what we were trying to avoid with the namespace prefix?

Reply:
We want to make sure that the resources themselves are name spaced. But, there's only so many valid GCP [regions](https://cloud.google.com/compute/docs/regions-zones/).

(Despite what the old gcp_region, gcp_zone, might've implied, you can't create new regions or zones)

Reply:
Updated.